chain/client-primitives/src/debug.rs

```diff
@@ -47,17 +47,33 @@ pub struct DebugChunkStatus {
 #[derive(Serialize, Deserialize, Debug)]
 pub struct DebugBlockStatus {
     pub block_hash: CryptoHash,
+    pub prev_block_hash: CryptoHash,
     pub block_height: u64,
+    pub block_timestamp: u64,
     pub block_producer: Option<AccountId>,
+    pub is_block_missing: bool, // only header available
```

Comment:
nit: could we rename this to "full_block_missing"? If I see this name without the comment,  I would have thought it could mean that the the block is missing on the chain.

Reply:
Yep!